ci/builders/linux_unopt.json

```diff
@@ -269,6 +269,100 @@
                     ]
                 }
             ]
+        },
+        {
+            "cas_archive": false,
+            "drone_dimensions": [
+                "device_type=none",
+                "os=Linux"
+            ],
+            "gclient_variables": {
+                "use_rbe": true
+            },
+            "gn": [
+                "--target-dir",
+                "ci/android_jit_release_x86_test",
+                "--android",
+                "--android-cpu=x86",
+                "--runtime-mode=jit_release",
+                "--rbe",
+                "--no-goma"
+            ],
+            "name": "ci/android_jit_release_x86_test",
+            "description": "Produces jit-release mode artifacts to target x86 Android from a Linux host.",
+            "ninja": {
+                "config": "ci/android_jit_release_x86_test",
+                "targets": [
+                    "flutter",
```

Comment:
so we're duplicating the compilation step here so that we can test from a different build?

Reply:
Yes. Compilation is relatively cheap since it's likely to be rbe cache hits.

Reply:
SGTM

Reply:
I'm just trying to build a mental model, since I know at some places we use CAS to share artifacts.

Reply:
Would it be fair to say that, were someone in the future to refactor the organization of all of this, these tests should depend on the compilation build, and get the artifacts via CAS?

Reply:
Maybe. For the engine, the cases where that is an optimization might be a bit limited. AFAIU, it's only an improvement when there are multiple test suites we'd like to run in parallel on different machines.